express/routes.js

```diff
@@ -213,6 +214,14 @@ function setupRoutes(ext) {
 
     FdkRoutes.post("/fp/uninstall", async (req, res, next) => {
         try {
+            const reqSignature = req.headers['x-ext-signature'];
```

Comment:
Can we have this validation at one common place as it is being used by webhooks also @jinalviranii